selectContact.d.ts

```diff
@@ -25,6 +25,7 @@ export interface AddressEntry {
 }
 
 export interface Contact {
+    recordID: string,
```

Comment:
Can you change casing to `recordId`?  I'll merge and release today if so.  Thanks!

Reply:
I have already done it. Thank you!